### ISSUE TITLE
Re-add MaxLineLength parameter for IfUnlessModifier

### ISSUE THREAD

**Issue by johnnyshields** [rubocop/rubocop#14657]
I'd like `IfUnlessModifier` could only take effect on very short cases, e.g. 25 chars are less.

It seemed there used to be `MaxLineLength` parameter which did this, but it was removed, and now we are forced to use `Layout/LineLength` with no way to override it.

I am currently disabling `Layout/LineLength` disabled, so this means Rubocop will *always* convert if/else to a modifier

To illustrate:

```ruby
# Short - GOOD, ok to always make modifier
raise(ArgumentError) if name.blank?

# Long - GOOD
if current_user && Airline::CheckInService.new(current_user, booking_reference: params[:booking_reference])&.success?
  redirect_to boarding_pass_path(params[:booking_reference])
end

# Long - BAD
redirect_to boarding_pass_path(params[:booking_reference]) if current_user && Airline::CheckInService.new(current_user, booking_reference: params[:booking_reference])&.success?
```

It seems this was raised previously:
- https://github.com/rubocop/rubocop/issues/6149
- https://github.com/rubocop/rubocop/issues/6077

**Comment by bbatsov** [comment]
Closing as stale — no engagement. The `MaxLineLength` parameter was intentionally removed. If there's a compelling use case, please open a fresh issue with concrete examples.
